core/templates/dev/head/css/oppia.css

```diff
@@ -3819,16 +3819,26 @@ md-card.preview-conversation-skin-supplemental-card {
   background-color: #fff;
   cursor: pointer;
   display: inline-block;
-  height: inherit;
+  height: 321px;
```

Comment:
Why these changes? I'm concerned it'll break other parts, like the library, creator dashboard and profile page -- have you tested?

Reply:
Done